ResearchKit/Common/ORKTaskViewController.m

```diff
@@ -981,10 +1000,6 @@ - (void)collectToolbarItemsFromViewController:(UIViewController *)viewController
     _pageViewController.toolbarItems = viewController.toolbarItems;
     _pageViewController.navigationItem.leftBarButtonItem = viewController.navigationItem.leftBarButtonItem;
     _pageViewController.navigationItem.rightBarButtonItem = viewController.navigationItem.rightBarButtonItem;
-    if (![self shouldDisplayProgressLabel]) {
-        _pageViewController.navigationItem.title = viewController.navigationItem.title;
-        _pageViewController.navigationItem.titleView = viewController.navigationItem.titleView;
-    }
```

Comment:
Why move the code? I think we should make sure `viewController.navigationItem.title` changing can trigger `collectToolbarItemsFromViewController`